src/error/syntaxError.js

```diff
@@ -33,6 +34,25 @@ export function syntaxError(
 }
 
 /**
+ * Produces a string for the invarant(...) function that renders the location
```

Comment:
Maybe we can even refactor these functions to be more generic:
```js
export function formatError(errorType: string) { ... }
export function syntaxError() { return formatError('syntax'); }
export function validationError() { return formatError('validation'); }
```
Not exactly like that, but you get the gist